source/Session.cpp

```diff
@@ -434,17 +435,12 @@ void Session::accumulateTrajectories()
 {
 	if (notNull(m_logger) && m_config->logger.logTargetTrajectories) {
 		for (shared_ptr<TargetEntity> target : m_targetArray) {
-			if (!target->isLogged()) continue;
-			// recording target trajectories
-			Point3 targetAbsolutePosition = target->frame().translation;
-			Point3 initialSpawnPos = m_camera->frame().translation;
-			Point3 targetPosition = targetAbsolutePosition - initialSpawnPos;
-					   
+			if (!target->isLogged()) continue;					   
```

Comment:
This change should be validated to function the same. Is there a currently existing test to make sure this isn't a regression?

Reply:
This particular line isn't a change, but I assume the reference is to the camera relative vs absolute coordinate change below? If so this is _not_ intended to function the same. Per previous discussion targets are now logged in world-space as opposed to relative to the camera frame

Reply:
Okay, I realize I was looking at this in the wrong context. Looks good.